src/gui/editors/SongEditor.cpp

```diff
@@ -124,9 +124,10 @@ void positionLine::setLineColor( const QColor & c )
 // in an unexpected location when positioned at the start of the track
 void positionLine::zoomChange( double zoom )
 {
-	move( x() + width() - 1, y() );
+	int m_x = x() + width() - 1;
+	
 	resize( 8.0 * zoom, height() );
-	move( x() - width() + 1, y() );
+	move( m_x - width() + 1, y() );
```

Comment:
The `m_` prefix is reserved for mutable class fields. Since this is just a function-scope variable, it shouldn't have the prefix.

Reply:
Renamed to `X` (would let me name it `x` as there is a function with that name).